packages/coachmark/src/CoachIndicator.ts

```diff
@@ -0,0 +1,42 @@
+/*
+Copyright 2023 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+import {
+    CSSResultArray,
+    html,
+    SpectrumElement,
+    TemplateResult,
+} from '@spectrum-web-components/base';
+import { property } from '@spectrum-web-components/base/src/decorators.js';
+import styles from './coach-indicator.css.js';
+
+/**
+ * @element sp-coach-indicator
+ */
+export class CoachIndicator extends SpectrumElement {
+    public static override get styles(): CSSResultArray {
+        return [styles];
+    }
+
+    @property({ type: Boolean, reflect: true })
+    public quiet = false;
+
+    @property({ reflect: true })
+    public variant: 'dark' | 'light' | '' = '';
```

Comment:
From a normalization perspective, does it make sense to make this `static="white"` and `static="black"` like we do in `<sp-action-button>` to satisfy https://opensource.adobe.com/spectrum-css/actionbutton.html#staticwhite

Reply:
To make this we need a updated css from the css team for static="white" and static="black" variant.
I think we should do this in a separate PR?

Reply:
You can do it today in the `spectrum-config.js`. Doing it later would be a freaking change to the API. It’s a new component that no one uses, so maybe that doesn’t matter as much as it does with the packages. Breaking changes will cost more when we get past 1.0, so getting a handle on when/why/etc now is good practice.

Reply:
Should be done now